diff --git a/ros/catkin/cmake/tools/rt.cmake b/ros/catkin/cmake/tools/rt.cmake
index 3e5b708..c12173b 100644
--- a/ros/catkin/cmake/tools/rt.cmake
+++ b/ros/catkin/cmake/tools/rt.cmake
@@ -31,7 +31,7 @@
 # message("CMAKE_SYSTEM_LIBRARY_PATH: ${CMAKE_SYSTEM_LIBRARY_PATH}")
 # message("CMAKE_VERSION=${CMAKE_VERSION}")
 
-if(NOT (APPLE OR WIN32 OR MINGW OR ANDROID))
+if(NOT (APPLE OR WIN32 OR MINGW OR ANDROID OR UNIX))
   if (${CMAKE_VERSION} VERSION_LESS 2.8.4)
     # cmake later than 2.8.0 appears to have a better find_library
     # that knows about the ABI of the compiler.  For lucid we just
